GettingStartedGuide.md

```diff
@@ -45,6 +45,7 @@ Once completed, one can progress to the [Use Security Features](UseSecurityFeatu
 - WiFi access point with access to the internet.
 
 ### 1.2 Software Requirements
+> **NOTE:** For convenience sake, there is an esp-idf v5.1.1 submoduled at the root of the directory, other versions on esp-idf may also be used but does not guarantee full compatibility.
```

Comment:
If we're going to add this submodule we should change the instructions to tell users to run the install and export scripts

Reply:
Sounds like a good idea, but a link to latest esp-idf document might also makes sense. what you think?